src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -106,6 +107,19 @@ uct_rc_mlx5_ep_inline_post(uct_rc_mlx5_ep_t *ep, unsigned opcode,
                                                       MLX5_WQE_CTRL_CQ_UPDATE);
         break;
 
+    case MLX5_OPCODE_SEND|UCT_RC_MLX5_OPCODE_FLAG_RAW:
+        /* Send empty AM with just AM id (used by FC) */
+        wqe_size         = sizeof(*ctrl) + sizeof(*inl) + sizeof(*rc_hdr);
+        UCT_CHECK_LENGTH(wqe_size, UCT_RC_MLX5_MAX_BB * MLX5_SEND_WQE_BB,
```

Comment:
no need to check length here since this is internal control message and not coming from API